api/docs/v2/storefront/index.yaml

```diff
@@ -49,6 +49,31 @@ paths:
           $ref: '#/components/responses/403Forbidden'
       security:
         - bearerAuth: []
+    post:
+      description: >-
+        Creates new account
+      tags:
+        - Account
+      operationId: 'Account Creation'
+      parameters:
```

Comment:
@chhga you need to use requestBody not parameters - https://swagger.io/docs/specification/describing-request-body/